chore/llcppg/ast/ast.go

```diff
@@ -218,10 +218,12 @@ func (*CommentGroup) exprNode() {}
 // ------------------------------------------------
 
 type Field struct {
-	Doc     *CommentGroup // associated documentation; or nil
-	Type    Expr          // field/method/parameter type; or nil
-	Names   []*Ident      // field/method/(type) parameter names; or nil
-	Comment *CommentGroup // line comments; or nil
+	Type     Expr            // field/method/parameter type; or nil
+	Names    []*Ident        // field/method/(type) parameter names; or nil
+	IsStatic bool            // static field
+	Comment  *CommentGroup   // line comments; or nil
+	Doc      *CommentGroup   // associated documentation; or nil
```

Comment:
don't change this.

Reply:
Updated, I misunderstood the new line here